XenAdmin/Wizards/NewNetworkWizard.cs

```diff
@@ -209,7 +209,7 @@ private DialogResult ShowSriovCreationWarning()
             var dlg = new ThreeButtonDialog(
                 new ThreeButtonDialog.Details(
                     SystemIcons.Warning, 
-                    Messages.SRIOV_NETWORK_CREATE_WILL_DISTURB_CONNECTION),
+                    Messages.SRIOV_NETWORK_CREATE_WARNING),

```

Comment:
Creating and launching dialogs should be done within a `using` block so we don't leak resources.